tooling/cli/Cargo.toml

```diff
@@ -64,6 +64,8 @@ ignore = "0.4"
 ctrlc = "3.2"
 log = { version = "0.4.17", features = [ "kv_unstable", "kv_unstable_std" ] }
 env_logger = "0.9.0"
+icns = { git = "https://github.com/FabianLars/rust-icns", rev = "bc2e492328cc5415a23a5eb6ec25632e8326f7bd" }
```

Comment:
We'll need to either publish this crate or move it to a module inside cli.rs

Reply:
Yes absolutely, that was mainly to show chip (and you) what I was talking about.
Aaand as discussed on discord I'm not really sure how to proceed with the crate (not happy with it for compressed pngs etc, but current state would be fine for 1.1)

Reply:
Let's go with `fine for 1.1` for now :P

Reply:
I guess you'd prefer to have that crate part of the tauri org? If so i'd just move my fork to the org and let you handle the publishing 😂

Reply:
Yeah :D